test/test_sparse.py

```diff
@@ -3427,6 +3447,7 @@ def test_cuda_sparse_cpu_dense_add(self):
         with self.assertRaisesRegex(RuntimeError, "add: expected 'self' to be a CUDA tensor, but got a CPU tensor"):
             x + sparse_y
 
+
```

Comment:
This change is probably unintentional.

Reply:
yes, it is left over from the first commit where the test was here instead. Reverting